src/test/scala/no/ndla/articleapi/service/converters/HTMLCleanerTest.scala

```diff
@@ -28,9 +30,76 @@ class HTMLCleanerTest extends UnitSuite {
   test("That HTMLCleaner removes comments") {
     val initialContent = LanguageContent(nodeId, nodeId, """<article><!-- this is a comment --><h1>heading<!-- comment --></h1></article>""", Some("en"))
     val expectedResult = "<article><h1>heading</h1></article>"
-    val (result, status) = HTMLCleaner.convert(initialContent, ImportStatus(Seq(), Seq()))
+    val (result, status) = HTMLCleaner.convert(initialContent, defaultImportStatus)
 
     result.content.replace("\n", "") should equal (expectedResult)
     result.requiredLibraries.length should equal (0)
   }
+
+  test("ingress is extracted when wrapped in <p> tags") {
```

Comment:
Konverterene er orgranisert i forskjellige moduler, der alle konverter modulene satt sammen former en "converter pipeline" (`ComponentRegistry.scala:101`).

Testene i HTMLCleanerTest.scala tester all funksjonalitet i kun denne konverter-komponenten.
Testene i ConverterServiceTest.scala tester hele pipelinen, for å forsikre at alle modulene som skal være lenket inn i pipelinen blir kjørt. Den tester da ikke funksjonaliteten i de forskjellige komponentene like nøye, fordi det blir testet i de individuelle converter-module komponentene (e.g HTMLCleanerTest.scala)